warehouse/README.md

```diff
@@ -0,0 +1,15 @@
+Welcome to your new dbt project!
```

Comment:
Should we delete this file, or at least make it relevant to our project? (Ex. at least reference `poetry run dbt run`)

Reply:
Hm yeah let me add some actually useful content